cfgmgr/coppmgr.cpp

```diff
@@ -250,7 +253,65 @@ void CoppMgr::mergeConfig(CoppCfg &init_cfg, CoppCfg &m_cfg, std::vector<std::st
         {
             std::vector<FieldValueTuple> cfg_fvs;
             cfgTable.get(i, cfg_fvs);
-            m_cfg[i] = cfg_fvs;
+            helper[i] = cfg_fvs;
```

Comment:
mergeConfig is used to merge both copp_trap and copp_group cfg. Its a generic function. I don't recommend to have the helper config present inside this API. The helper needs to be done only for copp group and not for copp trap.

Reply:
Moved to a dedicated function to merge with preserved copp tables.